src/plugins/intel_cpu/tests/functional/single_layer_tests/group_convolution.cpp

```diff
@@ -196,12 +200,51 @@ class GroupConvolutionLayerCPUTest : public testing::WithParamInterface<groupCon
                 ngraph::builder::makeGroupConvolution(paramOuts[0], netType, kernel, stride, padBegin,
                                                       padEnd, dilation, padType, convOutChannels, numGroups));
         function = makeNgraphFunction(netType, params, groupConv, "groupConvolution");
+        if (inputShape.second[0][1] / numGroups <= 16) {
+            isSmallInputChannel = true;
+        }
     }
 };
 
 TEST_P(GroupConvolutionLayerCPUTest, CompareWithRefs) {
     SKIP_IF_CURRENT_TEST_IS_DISABLED()
 
+   // Skip tests for brgconv convolution where kernel size = 1x1
+    if (priority[0] == "brgconv_avx512" || priority[0] == "brgconv_avx512_amx") {
+        bool is_1x1 = true;
+        for (const auto &i : kernel) {
+            if (i != 1) {
+                is_1x1 = false;
+                break;
+            }
+        }
+        if (is_1x1) {
+            GTEST_SKIP() << "Disabled test due to the brgconv does not support 1x1 convolution kernel." << std::endl;
+        }
+        if (isSmallInputChannel) {
+            GTEST_SKIP() << "Disabled test due to the brgconv does not support small input channel GroupConvolution on amx." << std::endl;
+        }
+    }
+    if (priority[0] == "brgconv_avx512" && isBF16) {
+        if (!(with_cpu_x86_bfloat16() && with_cpu_x86_avx512_core_vnni())) {
```

Comment:
According to the definition 'AVX2 < AVX512_CORE < AVX512_CORE_VNNI < AVX512_CORE_BF16 < AVX512_CORE_AMX' maybe we can simplify it to `if ((with_cpu_x86_bfloat16())`.

Reply:
I am not sure, I add this condition according to the code [jit_generator.cpp](https://github.com/openvinotoolkit/openvino/blob/af99d52ee464ec77759926714eadd796d4927d97/src/core/reference/src/runtime/reference/jit_generator.cpp#L67), where is the definition？does it mean if the CPU support "AVX512_CORE", the CPU must support "AVX2" ?

Reply:
Yes, here is the [definition](https://github.com/oneapi-src/oneDNN/blob/df54ea1ea850381129111987f06c61cb61a9a285/doc/performance_considerations/dispatcher_control.md).

Reply:
@luo-cheng2021 thank you very much, I still think need to use two conditions here, the condition "with_cpu_x86_bfloat16()" only check BF16 flag, "AVX512_CORE_BF16" contain two flag "AVX512_CORE_VNNI" and BF16 flag

Reply:
@luo-cheng2021 I update the code, and separate the instances, please help to review